tiledb/cloud/sql/util.py

```diff
@@ -0,0 +1,15 @@
+# This file includes the datatype mapping from pandas to the types required by the Python DB API 2.0
+import pyarrow as pa
+
+
+def getDBType(dtype):
```

Comment:
This can also probably be put in the same file as the two classes, and is probably better named `_get_db_type`.